example/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link href="ext/codemirror.css" rel="stylesheet" type="text/css">
```

Comment:
Yeah. I wouldn't take anything from that demo as an example of what to do.

I'm not sure we should be checking in CodeMirror sources, but let's see what the folks think on Gitter.